configs/client/supervisord.conf

```diff
@@ -24,5 +24,6 @@ stderr_logfile=syslog
 command = /usr/bin/veth-create.sh
```

Comment:
do we still need this script in the client docker?

Reply:
No, we don't

Reply:
Please remove then

Reply:
Not really. It is disabled.